includes/gateway/class-omise-payment-alipayplus.php

```diff
@@ -0,0 +1,158 @@
+<?php
+defined( 'ABSPATH' ) or die( 'No direct script access allowed.' );
```

Comment:
Not sure about PHP convention, I noticed there are spaces between parentheses  `defined( 'ABSPATH' )`. Is that the PHP conventional?

Reply:
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage

It's kind of a wordpress convention kub and some other PHP frameworks follow it too, I think it's meant to make PHP easier to read